src/socket.rs

```diff
@@ -200,6 +200,9 @@ impl Socket {
     /// non-blocking mode before calling this function), socket option can't be
     /// set *while connecting*. This will cause errors on Windows. Socket
     /// options can be safely set before and after connecting the socket.
+    ///
+    /// On Cygwin, a Unix domain socket connect blocks until the server accepts
+    /// it. If the behavior is not expected, try `Socket::set_no_peercred`.
```

Comment:
Let's have working links. (here and `accept`)
```suggestion
    /// On Cygwin, a Unix domain socket connect blocks until the server accepts
    /// it. If the behavior is not expected, try [`Socket::set_no_peercred`].
```

Reply:
Oh, no. `set_on_peercred` is only available on cygwin, so rustdoc warns on other platforms.

Reply:
You might be able to use `#[cfg(any(docsrs, ...))]` to show the function when building docs for Linux.